meta/SchemaBuilder.php

```diff
@@ -199,6 +202,26 @@ protected function updateColumns() {
         return true;
     }
 
+    private function migrateSingleToMulti($table, $colref) {
+        $sqlSelect = "SELECT pid, rev, col$colref AS value FROM data_$table WHERE latest = 1";
+        $res = $this->sqlite->query($sqlSelect);
+        $valueSet = $this->sqlite->res2arr($res);
+        $this->sqlite->res_close($res);
+        $valueString = array();
+        foreach ($valueSet as $values) {
```

Comment:
Under what circumstances can the select above return more than one row? There should be only one ``latest`` entry for one column in a data_* table!?